src/validation/error.messages.ts

```diff
@@ -197,6 +197,7 @@ export enum ErrorMessages {
   SELECT_IF_ANY_TRUSTS_INVOLVED = "Select yes if there are any trusts involved in this overseas entity",
   SELECT_DO_YOU_WANT_TO_MAKE_OE_CHANGE = "Select yes if you need to make any changes to this overseas entity",
   SELECT_DO_YOU_WANT_TO_MAKE_CHANGES_UPDATE_STATEMENT = "Select yes if all the information in this update statement is correct",
+  SELECT_DO_YOU_WANT_TO_MAKE_CHANGES_REMOVE_STATEMENT = "Select yes if all the information in this application correct",
```

Comment:
Guessing this text relates to AC3 but I can't see this actual text on [this](https://companieshouse.atlassian.net/wiki/spaces/ROE2/pages/4334977098/Error+messages) page and if it is correct, it looks like there's an "is" missing from the near the end.

Reply:
one of the messages was missing from the guide, so we changed it based on the other text changes, eg, converting 'update statement' to 'application'.

The 'is' should be added so we can fix that.

Reply:
Ok. I think we need to get AC updated though (otherwise this might come back as a defect/tester query).